### DISCUSSION TITLE
Swift -> ARM64 Support?

### DISCUSSION THREAD

**Discussion by meltcow** [compiler-explorer/compiler-explorer#4400] [General]
Hello, everybody! I have a question.

I want to compile Swift into ARM64, is this possible to do with CE? I see that it isn't supported by default, but is this something that any of you kind folks know something about and can help me to do? I'm very curious about exploring the Swift -> ARM Assembly relationship, as easily as possible. 

Thank you!

Regards,

GS

**Comment by partouf** [comment]
The Swift compilers that we download from the website @ https://www.swift.org/download/ do not seem to be able to cross-compile. So this isn't something that we can easily made to work. Perhaps when someday we'll add ARM instances to our fleet, but we currently have no solid plans for that yet.
